examples/python3/Dockerfile

```diff
@@ -1,8 +1,14 @@
-FROM python:3-slim AS build-env
+FROM python:3.6-stretch AS build-env
 ADD . /app
 WORKDIR /app
+RUN pip3 install --upgrade pip
+RUN pip3 install -r requirements.txt
```

Comment:
(I am not versed in Python.) This directory does not contain `requirements.txt`. Does this work OK without the file?

Reply:
Many assumptions here, yes. see the link in opening comment for context. Long term I suspect we need an actual example .py file for this demo along with tons of comments in the Dockerfile.

so adding everything from current project directory to the /app, change dir to the /app then use the projects requirements.txt

Reply:
Sorry, I didn't get it. The example in the opening comment does have the `requirements.txt`: https://lathama.net/git/lathama/Dockerfiles/src/branch/master/distroless/cherrypyexample/requirements.txt

However, as you can see, our Distroless repo example doesn't have the file: https://github.com/GoogleContainerTools/distroless/tree/master/examples/python3

Can you confirm `docker built .` on this example won't fail on this `pip3 install -r requirements.txt` line?

Reply:
I understand what you are asking. I will work up a more complete example later this week.